sprint-docs/09-demo-plan.md

```diff
@@ -0,0 +1,43 @@
+
+# Bookit Iteration #9: Demo Plan
+
+Demo date: Wednesday July 26, 2017
+
+1) User notification when booking fails (#7)
+2) Prevent users from attempting to make bookings in the past (#8)
+3) Current time indicator (yellow line) appears only when viewing today's agenda (#9)
+4) Edit bookings (#6)
+
+## 1 - Failed bookings
+1) See the agenda view, set to today's date.
+2) Click on the timeline in an available slot and enter a title for the booking.
+3) Disable the network in the browser.
+4) See messaging indicating that the booking failed.
+
+## 2 - Booking in the past
+### Clicking on the timeline
+1) See the agenda view, set to today's date.
+2) Click on the timeline to the left of the yellow time indicator.
+3) See that the meeting form does not appear.
+
+### Changing the time in the form
+1) See the agenda view, set to today's date.
+2) Click on the timeline to the right of the yellow time indicator.
+3) Change the start/end time to a time or date in the past.
+4) See a message in the left pane indicating that the booking must be in the future.
+
+## 3 - Current time indicator
+1) See the agenda view, set to today's date.
+2) See the yellow line, indicating the current time.
+3) Click on a past or future date in the calendar.
+4) See the agenda view, displaying all meetings, with no yellow line to indicate the current time.
+
+## 4 - Editing a booking
```

Comment:
do we want to demo the edit button being missing when already editing a meeting?